### PR TITLE
fix #39369 thelibertydaily.com

### LINKED ISSUE #39369: thelibertydaily.com
### Issue URL (Ads)
[https://thelibertydaily.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fthelibertydaily.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.2.1
Browser: | Firefox
Filters: | Filter unblocking search ads and self-promotion,<br>AdGuard Base
Other extensions used: | uBlock Origin and AdBlocker Ultimate

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/39369

### CODE REVIEW

File: EnglishFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -578,6 +578,7 @@ rarbgproxied.org,rarbg.to,rarbg-to.proxydude.xyz,rarbg-to.pbproxy.red,rarbg.unbl
 alluc.ee#?##resultlist > div#resultitems[-ext-has="> div"]~iframe:not([src])
 !
 spankwire.com#?##js-react-video-player > div > div > div:has(> a[href="https://www.czechsexcasting.com/"])
+thelibertydaily.com##.widget-box:has(> div.widget-text > div.powerinbox > div.pi_header)
```
Comment (BlazDT) [inline EnglishFilter/sections/css_extended.txt]: #?#

File: EnglishFilter/sections/content_blocker.txt [hunk 2]
```diff
@@ -500,6 +500,8 @@ aioboot.com,dist-tutor.info,investmentwatchblog.com,life.ru,1000ps.de,abw.by,abw
 ! https://github.com/AdguardTeam/AdguardFilters/issues/39571
 razorlink.us#@#.adsbygoogle
 @@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=razorlink.us
+! https://github.com/AdguardTeam/AdguardFilters/issues/39369
+||thelibertydaily.com/*/adtoniq/js/niqon.js
```
Comment (BlazDT) [inline EnglishFilter/sections/content_blocker.txt]: Not seeing adblock detection.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/css_extended.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/css_extended.txt]: Fixed, thanks.

File: EnglishFilter/sections/content_blocker.txt (hunk 2 above)
Comment (zloyden) [reply EnglishFilter/sections/content_blocker.txt]: Well, on my iPhone I can't reproduce it too. Rule was deleted.

**Review by BlazDT — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**